Source/param_includes/maestro_defaults.H

```diff
@@ -42,11 +44,15 @@ bool        Maestro::plot_base = false;
 bool        Maestro::plot_gpi = true;
 bool        Maestro::plot_cs = false;
 std::string Maestro::plot_base_name = "plt";
+std::string Maestro::small_plot_base_name = "smallplt";
 std::string Maestro::check_base_name = "chk";
 int         Maestro::diag_buf_size = 10;
 bool        Maestro::plot_ad_excess = false;
 bool        Maestro::plot_processors = false;
 bool        Maestro::plot_pidivu = false;
+#ifdef magvel"
```

Comment:
I generated the new `param_includes` updates by running the `mk_params.sh` script - any ideas why it inserted these ifdefs around the `small_plot_vars` variable?